content.js

```diff
@@ -1,17 +1,32 @@
-if ($(".fullScreenButton.enter").length == 1){
-	$(".fullScreenButton").click();
-	$(".tray.screencast").hide();
-	$(".rightSide").hide();
-	$(".overlay").hide();
-	$(".normalControls").hide();
-	$(".controls").hide();
+var vetoed = [".roomControlsLayoutView", ".overlay", ".normalControls", ".rightSide", ".tray.screencast", ".social", ".toolbar", ".barRight", ".controls", ".tray"];
+
+if ($(".controls").is(":visible")){
```

Comment:
Do you think this IF needs to be improved? If they manually exit fullscreen, and they use the shortcut would there be an issue on how the cleaner works?

Reply:
Ideally we would be the only ones modifying these elements. Thus, if the user manually exits full screen and toggles the cleaning again, rabbit cleaner will think that rabbit has already been cleaned and will proceed to unclean rabbit. I believe this should be the expected behaviour since they are other faster shortcuts if they simply wanted it to be full screened again.

Reply:
That's valid reasoning. I think it's fine as is then.